RobotNavigation/Maze.cs

```diff
@@ -9,69 +9,64 @@ namespace RobotNavigation
 {
     public class Maze
     {
-        public enum Cell
-        {
-            Empty,
-            Target,
-            Wall
-        }
+        
 
         public int Width { get; set; }
         public int Height { get; set; }
        
-        private Cell[,] _maze;
+        private Position.Cell[,] _maze;
 
         private ArrayList _targets;
-        private Position _pos;
+        private Position _target;
 
         public Maze()
         {
-            _pos = new Position();
+            _target = new Position();
             _targets = new ArrayList();
         }
 
         public Maze(int height, int width)
         {
             Width = width;
             Height = height;
-            init();
+            Init();
         }
 
-        public void init()
+        public void Init()
         {
-            _maze = new Cell[Width, Height];
+            _maze = new Position.Cell[Width, Height];
             for (int i = 0; i < Width; i++)
             {
                 for (int j = 0; j < Height; j++)
                 {
-                    _maze[i, j] = Cell.Empty;
+                    _maze[i, j] = Position.Cell.Empty;
                 }
             }
         }
         
-        public Cell getCell(int x, int y)
+        public Position.Cell GetCell(int x, int y)
```

Comment:
Not sure what this method is for now.  A Position needs to have the following attributes:
- An X coordinate
- A Y coordinate
- A CellType (enum) - which is a wall, a space, or a target.  

If a Position contains all of the above, i'm not sure you then need the list of targets in the Maze object.  It will have that information stored in the two dimensional array of Position objects.

Reply:
What do I change GetCell to in the MazeView class?

Reply:
GetPosition